### ISSUE TITLE
Add the ability to expand comparison to support use cases where you want to check if uptime is greater than.

### ISSUE THREAD

**Issue by juan-moreno** [sensu-plugins/sensu-plugins-uptime-checks#9]
Some machines need to be checked if uptime is upper than x seconds, eg in our case we are using dynamic Jenkins slaves (with EC2 plugin) and we want to know if Jenkins slave is up for a long time.

**Comment by majormoses** [comment]
I think we would be better served by an `--operator` option to specify the comparison. Much more logical to say `chec-uptime.rb --warn 180 --operator gt` We would default to `lt` to keep backwards compatibility. We should also restric the parameters to something sensible like this:
```
  option :operator,
         short: '-o OPERATOR',
         long: '--operator OPERTATOR',
         description: 'operator to do comparison.',
         default: lt
         in: %w(lt gt le ge eq)
```

**Comment by juan-moreno** [comment]
Hi @majormoses,

Thanks for propose an alternative approach and in a lot of case I would be agree with you but in this particular case (machine uptime) comparations like ge, le or eq has no sense, only lt and gt are needed. 

From the point of view of someone that use the check is more easy to think in an option that reverse the normal behavior than think as a developer using operators, besides using operators approach add innecesary complexity in the code.

Thank you very much for  thinking in other ways to treat the same issue, this absolutely enrich.

**Comment by majormoses** [comment]
I see, I still think I prefer exposing the comparison operator (we could limit it to lt and gt if we want) but I do take your point. I will put my review comments on how we could make it a bit better with the proposed approach.

**Comment by juan-moreno** [comment]
Hi @majormoses

Agree with your review I've changed the code to adapt to your suggestions.

**Comment by majormoses** [comment]
fixed via #10
